Gopkg.lock

```diff
@@ -0,0 +1,75 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/beevik/etree"
+  packages = ["."]
+  revision = "15a30b44cfd6c5a16a7ddfe271bf146aaf2d3195"
+  version = "v1.0.0"
+
+[[projects]]
+  name = "github.com/crewjam/saml"
```

Comment:
Other than watching a gophercon talk about it, I've not used `dep`, so forgive a bit of ignorance here. Is it right to have our own package in the list?

Reply:
Good catch, that happened because I was running it from github.com/sevoma/saml folder. When run from crewjam/saml, it does not appear and works properly.